src/Microsoft.ML.StandardLearners/Standard/SdcaMultiClass.cs

```diff
@@ -57,41 +57,36 @@ public sealed class Arguments : ArgumentsBase
         /// <param name="l2Const">The L2 regularization hyperparameter.</param>
         /// <param name="l1Threshold">The L1 regularization hyperparameter. Higher values will tend to lead to more sparse model.</param>
         /// <param name="maxIterations">The maximum number of passes to perform over the data.</param>
-        /// <param name="advancedSettings">A delegate to set more settings.
-        /// The settings here will override the ones provided in the direct method signature,
-        /// if both are present and have different values.
-        /// The columns names, however need to be provided directly, not through the <paramref name="advancedSettings"/>.</param>
         public SdcaMultiClassTrainer(IHostEnvironment env,
```

Comment:
>  public SdcaMultiClassTrainer [](start = 6, length = 30)

same

Reply:
made internal. deletion will happen in #2100

---
In reply to: [248770465](https://github.com/dotnet/machinelearning/pull/2093#discussion_r248770465) [](ancestors = 248770465)